django_celery_beat/admin.py

```diff
@@ -24,8 +24,8 @@ class TaskSelectWidget(Select):
 
     def tasks_as_choices(self):
         _ = self._modules
-        tasks = list(sorted(name for name in self.celery_app.tasks
-                            if not name.startswith('celery.')))
+        tasks = sorted(name for name in self.celery_app.tasks
```

Comment:
I'm not fully sure about this change

Reply:
It is unnecessary to use `list()` around `sorted()`, as the latter already returns a `list`.

Running the following on the commandline...

% [`ruff rule C413`](https://docs.astral.sh/ruff/rules/#flake8-comprehensions-c4)
# unnecessary-call-around-sorted (C413)

Derived from the **flake8-comprehensions** linter.

Fix is always available.

## What it does
Checks for unnecessary `list` or `reversed` calls around `sorted`
calls.

## Why is this bad?
It is unnecessary to use `list` around `sorted`, as the latter already
returns a list.

[ ... ]